src/Server/TCPHandler.cpp

```diff
@@ -208,6 +210,15 @@ void TCPHandler::runImpl()
             if (!receivePacket())
                 continue;
 
+            /** If Query received, then settings in query_context has been updated
```

Comment:
Does it open a scenario for DDoS attacks by overrunning the pool of connections with a very large poll intervals?

Reply:
Default timeout (3600) is also huge enough to create DDos, and also most of the time it is useful to decrease the value (for tests) not increase.

Also if someone don't want to allow increasing this value he can create constraints for this.

And BTW this timeout is only for idle connection, so you can create lots of connections with ignoring this setting anyway with something like this `select sleepEachRow(1) from system.numbers(10000) settings receive_timeout=86400` (or something like this).